### PR TITLE
Misc Windows CI stuff

### PR DESCRIPTION
**Pull request by ubitux**

### CODE REVIEW

File: .github/workflows/ci_msvc.yml [hunk 1]
```diff
@@ -59,5 +59,16 @@ jobs:
       - name: Python import and dylib runtime test
         run: |
           .\venv\Scripts\ngli-activate.ps1
+
+          "Dynamic loading information of the nopegl DLL:"
```
Comment (mbouron) [inline .github/workflows/ci_msvc.yml:63]: DLL -> library

File: .github/workflows/ci_mingw.yml [hunk 2]
```diff
@@ -38,5 +38,13 @@ jobs:
       - name: Python import and dylib runtime test
         run: |
           . venv/bin/ngli-activate
+
+          echo "Dynamic loading information of the nopegl library:"
```
Comment (mbouron) [inline .github/workflows/ci_mingw.yml:42]: Dynamic loading information -> Shared libraries required by

**Review by mbouron — APPROVED** [review]
Except from my comments, LGTM.

File: .github/workflows/ci_msvc.yml (hunk 1 above)
Comment (ubitux) [reply .github/workflows/ci_msvc.yml]: done

**Pull request merged**